src/cpptools.ts

```diff
@@ -431,7 +431,7 @@ export class CppConfigurationProvider implements cpptools.CustomConfigurationPro
         // For CppTools V6 and above, build the compilerFragments data, otherwise build compilerArgs data
         const useFragments: boolean = this.cpptoolsVersion >= cpptools.Version.v6;
         // If the file didn't have a language, default to C++
-        const lang = fileGroup.language === "RC" ? undefined : fileGroup.language;
+        const lang = fileGroup.language === "RC" || fileGroup.language === "Swift" ? undefined : fileGroup.language;
```

Comment:
Should this be an include list (for C, C++ and ASM) rather than an exclude list? Wouldn't the same issue happen for [other CMake-supported languages](https://cmake.org/cmake/help/latest/command/project.html)?

> Supported languages are C, CXX (i.e. C++), CSharp (i.e. C#), CUDA, OBJC (i.e. Objective-C), OBJCXX (i.e. Objective-C++), Fortran, HIP, ISPC, Swift, ASM, ASM_NASM, ASM_MARMASM, ASM_MASM, and ASM-ATT.